tests/test_decision_service.py

```diff
@@ -1356,11 +1349,9 @@ def test_get_variation_for_feature__returns_none_for_user_not_in_group(self):
         feature = self.project_config.get_feature_from_key("test_feature_in_group")
 
         with mock.patch(
-            "optimizely.decision_service.DecisionService.get_experiment_in_group",
-            return_value=[None, []],
-        ) as mock_get_experiment_in_group, mock.patch(
-            "optimizely.decision_service.DecisionService.get_variation"
-        ) as mock_decision:
+                "optimizely.decision_service.DecisionService.get_variation",
```

Comment:
This mock does not give a test for "group". It tests the cases for feature-tests not bucketed.
Can we mock bucket to control group-exclusion cases?